internal/jobs/twitter_test.go

```diff
@@ -12,53 +17,99 @@ import (
 	"github.com/masa-finance/tee-worker/api/types"
 	. "github.com/masa-finance/tee-worker/internal/jobs"
 	"github.com/masa-finance/tee-worker/internal/jobs/stats"
+	"github.com/masa-finance/tee-worker/internal/jobs/twitterx"
 )
 
+// parseTwitterAccounts parses TWITTER_ACCOUNTS environment variable like production does
+func parseTwitterAccounts() []string {
+	accountsEnv := os.Getenv("TWITTER_ACCOUNTS")
+	if accountsEnv == "" {
+		return nil
```

Comment:
[minor nit] Wouldn't it be better to `return []string{}`? That way we don't have to check for `nil` anywhere (and even possibly remove a bunch of `if len(accounts) == 0`). We should do this in production too (haven't seen the relevant code).

Reply:
captured in test and production (config.go)